src/interfaces/alteration-interface.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  BusinessIF, NameRequestIF, NameTranslationIF, ShareStructureIF, ContactPointIF
+} from '@/interfaces'
+
+/**
+ * A filing's alteration object from the API. See:
+ * https://github.com/bcgov/business-schemas/blob/master/src/registry_schemas/schemas/alteration.json
+ */
+export interface AlterationIF {
+  provisionsRemoved: boolean
+  business: BusinessIF
+  nameRequest?: NameRequestIF
+  nameTranslations: NameTranslationIF[]
+  shareStructure: ShareStructureIF
+  contactPoint: ContactPointIF
+  [propName: string]: any // excess properties
```

Comment:
This is require to "catch" any extra properties on the IF without causing a TS error.

For example, `alteration.severin` would be OK.

Ideally the IF would contain all and only the correct properties, but this allows extra properties to be added as needed without causing errors. That said, we _should_ go back and add legit properties to the IFs as we go along.

Reply:
I like it!